ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -390,6 +391,57 @@ impl<'a> Assemble<'a> {
         alloc_off
     }
 
+    /// Sign extend the `from_bits`-sized integer stored in `reg` up to the full size of the 64-bit
+    /// register.
+    ///
+    /// `from_bits` must be between 1 and 64.
+    fn sign_extend_reg64(&mut self, reg: Rq, from_bits: u8) {
```

Comment:
Should probably be `sign_extend_to_reg64`?

Reply:
Fixed in 53f3c3aa4f0ff0ad1a4cc524a5430646683a894a